Tests/SecureXPCTests/TestHelpers.swift

```diff
@@ -8,12 +8,18 @@
 import XCTest
 @testable import SecureXPC
 
-// MARK: Encoding entry point
+// MARK: Encoding/Decoding
 
 func encode<T: Encodable>(_ input: T) throws -> xpc_object_t {
 	try XPCEncoder.encode(input)
 }
 
+func decode<T: Decodable>(_ input: xpc_object_t) throws -> T {
+	let dummyDict = xpc_dictionary_create(nil, nil, 0)
+	xpc_dictionary_set_value(dummyDict, "key", input)
+	return try XPCDecoder.decode(T.self, from: dummyDict, forKey: "key")
```

Comment:
Only dictionaries can be decoded at the top level, so this needs a similar workaround to what the encoding side needed before. I can fix it once this is merged.

Reply:
Thanks to https://github.com/trilemma-dev/SecureXPC/commit/7f71ebb623be881c300c740a590af7d6f5ecbf61, this is now fixed :)